_posts/en/meetings/2018-06-28-meeting.md

```diff
@@ -0,0 +1,231 @@
+---
+title: IRC meeting summary for 2018-06-28
+lang: en
+permalink: /en/meetings/2018/06/28/
+name: 2016-06-28-meeting
+layout: page
+type: meetings
+version: 1
+---
+{% include _toc.html %}
+{% include _references.md %}
+
+- View this week's log on [BotBot.me][bbm log] or [MeetBot][mb log]
+- [Meeting minutes by MeetBot][mb minutes]
+
+---
+
+Topics discussed during this weekly meeting included what pull requests
+members of the project would like reviewers to focus on during the
+upcoming week, a draft specification for a new [BIP39][]-like seed
+format, documenting how Bitcoin Core uses the [BIP32][] HD wallet
+standard, an update on progress towards implementing [BIP151][]
+encrypted peer-to-peer connections, and discussion surrounding a new way
+to describe to wallet software what output scripts they should consider
+part of the user's wallet.
+
+## High priority for review
+
+**Background:** each meeting, Bitcoin Core developers discuss which Pull
+Requests (PRs) the meeting participants think most need review in the
+upcoming week.  Some of these PRs are related to code that contributors
+especially want to see in the next release; others are PRs that are
+blocking further work or which require significant maintenance (rebasing)
+to keep in a pending state.  Any capable reviewers are encouraged to
+visit the project's list of [current high-priority
+PRs][].
+
+**Discussion ([log][log hipri]):** the only PR specifically discussed
+was [#12196][], which adds a `scantxoutset` RPC to allow applications to
+search all current Unspent Transaction Outputs (UTXOs) for those that match a
+particular address, public key, private key, or HD wallet extended
+public key (xpub).
+
+Pieter Wuille suggested that the PR's author, Jonas Schnelli, remove
+xpub support from `scantxoutset` for now and Wuille would later PR an
+update to `scantxoutset` that adds support for the output script
+descriptors described later in these meeting notes.  If Wuille is
+delayed in doing so, Schnelli can just add the previously-written xpub
+support back in before the Bitcoin Core 0.17 feature freeze.  Schnelli
+agreed.
+
+*Note: extended discussion of output script descriptors during this part
+of the meeting has been moved to a [separate section](#output-script-descriptors).*
+
+## Cipherseed
+
+**Background:** (Part of the discussion.)
```

Comment:
nit: perhaps better to just omit this line?

Reply:
I'm not attached to it, but since every other meeting summary I've ever written always has a Background section for each topic, I thought it was a useful addition so that readers don't think it's accidentally missing.  It's basically the UI principle of least surprise applied to documentation.

Let me know if you still want it omitted and I'll edit.

Reply:
Fine either way. The wording just seemed strange to me, but fine to leave it in.